### PR TITLE
Fioefi feature scarthgap

### PR DESCRIPTION
**Pull request by igoropaniuk**

### CODE REVIEW

**Review by quaresmajose — APPROVED** [review]
LGTM but note I weekly rebase this branch

**Review by MrCry0 — APPROVED** [review]

File: meta-lmp-base/recipes-sota/fioefi/fioefi_0.1.bb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+SUMMARY = "OP-TEE Foundries.IO UEFI Firmware Update control script"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COMMON_LICENSE_DIR}/MIT;md5=0835ade698e0bcf8506ecda2f7b4f302"
+
+FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
```
Comment (ricardosalveti) [inline meta-lmp-base/recipes-sota/fioefi/fioefi_0.1.bb:5]: Not needed as this is not an append

Comment (quaresmajose) [reply meta-lmp-base/recipes-sota/fioefi/fioefi_0.1.bb:5]: I fixed this in a force push

File: meta-lmp-base/recipes-sota/fioefi/fioefi_0.1.bb [hunk 2]
```diff
@@ -0,0 +1,25 @@
+SUMMARY = "OP-TEE Foundries.IO UEFI Firmware Update control script"
```
[the remark this answers is not in the document]
Comment (quaresmajose) [reply meta-lmp-base/recipes-sota/fioefi/fioefi_0.1.bb:1]: I fixed this in a force push

**Pull request merged**